static/app/views/replays/detail/console/consoleLogRow.tsx

```diff
@@ -24,7 +24,8 @@ type Props = {
   onDimensionChange?: (
     index: number,
     path: string,
-    expandedState: Record<string, boolean>
+    expandedState: Record<string, boolean>,
+    event: MouseEvent<HTMLDivElement>
   ) => void;
```

Comment:
nit: this type could come from the new hook somehow. to save re-typing it each time